src/plugins/data_explorer/public/components/app_container.tsx

```diff
@@ -20,18 +20,38 @@ export const AppContainer = ({ view, params }: { view?: View; params: AppMountPa
 
   const { Canvas, Panel, Context } = view;
 
+  const MemoizedPanel = memo(Panel);
+  const MemoizedCanvas = memo(Canvas);
+
   // Render the application DOM.
   return (
     <EuiPage className="deLayout" paddingSize="none">
       {/* TODO: improve fallback state */}
       <Suspense fallback={<div>Loading...</div>}>
         <Context {...params}>
-          <Sidebar>
-            <Panel {...params} />
-          </Sidebar>
-          <EuiPageBody className="deLayout__canvas">
-            <Canvas {...params} />
-          </EuiPageBody>
+          <EuiResizableContainer>
```

Comment:
```suggestion
         <EuiResizableContainer direction={isMobile ? 'vertical' : 'horizontal'}>
```

And you can get the value of isMobile as follows:

```ts
import { ..., useIsWithinBreakpoints } from '@elastic/eui';
// ...
const isMobile = useIsWithinBreakpoints(['xs', 's', 'm']);
```